tests/test_lstm_policy.py

```diff
@@ -31,7 +38,32 @@ def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, n_lstm=64,
                          layer_norm=True, feature_extraction="mlp", **_kwargs)
 
 
+def _pos_obs(full_obs):
```

Comment:
maybe as a static method of CartPoleNoVelEnv it would make more sense?

Reply:
done